### DISCUSSION TITLE
dem issue

### DISCUSSION THREAD

**Discussion by 882010** [gmtsar/gmtsar#984] [Q&A]
I have been using this GMTSAR website to obtain DEM, but for the past two days, this issue has started appearing 
![Screenshot from 2024-08-28 10-30-26](https://github.com/user-attachments/assets/ea46b2f2-1e2f-499f-bb7b-1a585a1b0d67)
![Screenshot from 2024-08-28 10-30-26](https://github.com/user-attachments/assets/b2f5a52d-82b7-4839-a5ab-f6cc0fbc6c6c)
(Query could not be completed successfully . I have tried multiple times, but the problem persists. Is there a reason or a solution for this? I am stuck at this step.

**Comment by Xiaohua-Eric-Xu** [comment]
This website is currently down due to upgrade to RHL9 operating system. Meanwhile, you may use the make_dem.csh script in GMTSAR to directly download DEM from the GMT server (which should be very handy to do).
